src/framework/components/anim/component.js

```diff
@@ -475,6 +474,14 @@ class AnimComponent extends Component {
         }
     }
 
+    unbind() {
+        if (!this._normalizeWeights) {
```

Comment:
Given the function name `unbind`, it would imply it would unbind regardless of `_normalizeWeights`. Do we want the if check here or in the setter of `normalizeWeights`?

Reply:
We'd want this check any time unbind is called (currently changing the normalizeWeights property or removing the state graph).